.github/workflows/release.yml

```diff
@@ -0,0 +1,45 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - "v2.*"
+
+permissions:
+  contents: write # require for uploading releases
+
+env:
+  DEFAULT_GO_VERSION: ^1.18
+  GITHUB_USERNAME: ${{ secrets.EC2_BOT_GITHUB_USERNAME }}
+  GITHUB_TOKEN: ${{ secrets.EC2_BOT_GITHUB_TOKEN }}
+  WEBHOOK_URL: ${{ secrets.WEBHOOK_URL }}
+
+jobs:
+  release:
+    name: release
+    runs-on: ubuntu-20.04
```

Comment:
Slight preference for using `ubuntu-22.04` since it will be supported longer than 20.04. But I can also see an argument for keeping it the same as our other projects/branches.